scripts/internal/check_broken_links.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+# Author : Himanshu Shekhar < https://github.com/himanshub16 > (2017)
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
```

Comment:
Please copy the license you see in other (all) py files.
Also, this file should be executable (chmod +x file)